test/test_mode_app.c

```diff
@@ -0,0 +1,92 @@
+/* Wirepas Oy licensed under Apache License, Version 2.0
+ *
+ * See file LICENSE for full license details.
+ *
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "wpc.h"
+#include "platform.h"
+#include "test_mode.h"
+
+#define LOG_MODULE_NAME "test_mode_app"
+#define MAX_LOG_LEVEL INFO_LOG_LEVEL
+#include "logger.h"
+
+// Default serial port
+#ifdef PLATFORM_IS_WIN32
+static char * port_name = "COM22";
+#elif PLATFORM_IS_LINUX
+static char * port_name = "/dev/ttyUSB0";
+#elif PLATFORM_IS_DARWIN
+static char * port_name = "/dev/cu.usbmodem0001234567890";
+#else
+static char * port_name = "/dev/ttyS0";
+#endif
```

Comment:
Should it come later once the commit for multiple platform is merged?

Reply:
Will be fixed in next commit.